Marble Blast Platinum/platinum/client/defaults.cs

```diff
@@ -118,9 +118,9 @@
 $pref::recordingIndicator = 1;
 $pref::SearchHuntBonusDC = 1;
 
-$pref::warpSpeedMusic = 0;
-$pref::finalLapMusic  = 0;
-$pref::panicMusic     = 0;
+$pref::warpSpeedMusic = 1;
+$pref::finalLapMusic  = 1;
+$pref::panicMusic     = 1;
```

Comment:
I know we've had kinda a commit war over this but please revert this. We already did in ce6e510b6c35d3b787b8a567035a6a0c437bd1df and I really don't think we should bring it back